README.md

```diff
@@ -9,39 +9,98 @@ description: "A collection of samples showing how to use the Azure SDK for Go."
 
 # Azure SDK for Go Samples
 
-azure-sdk-for-go-samples is a collection of sample usages of the current version of the [Azure/azure-sdk-for-go][][here](https://github.com/Azure-Samples/azure-sdk-for-go-samples/tree/main/services). The current version of the [Azure/azure-sdk-for-go][] is referring the service packages under the `/services` directory.
+This repo contains a collection of sample usages of the new version of [Azure/azure-sdk-for-go][]. All the samples are placed under [`sdk`](https://github.com/Azure-Samples/azure-sdk-for-go-samples/tree/main/sdk) folder and the folder structures are corresponding to the service packages under the `/sdk` directory of the [Azure/azure-sdk-for-go][] repo.
```

Comment:
are we missing a url link here?

Reply:
Nope. It will use the link defined in the end of the doc.